elasticdl/python/common/k8s_client.py

```diff
@@ -175,6 +186,17 @@ def get_ps_service(self, ps_id):
             logger.warning("Exception when reading PS service: %s\n" % e)
             return None
 
+    def get_worker_service(self, worker_id):
+        try:
+            return self.client.read_namespaced_service(
+                # Worker service has the same name as pod name
+                name=self.get_worker_pod_name(worker_id),
```

Comment:
Although it is the same name, here we can use `get_worker_service_name(worker_id)`, right?

Reply:
Good call. Just updated the PR.